apiserver/facades/client/controller/controller.go

```diff
@@ -967,7 +967,7 @@ var runMigrationPrechecks = func(
 		}
 	}
 	err = client.Prechecks(ctx, modelInfo)
-	return errors.Annotate(err, "target prechecks failed 2")
+	return errors.Annotate(err, "target prechecks failed")
```

Comment:
I'm guessing the change to add "2" to the end of the message here was to distinguish this message from other identical message?

After a grep, I see:
```
$ grep -r "target prechecks failed"
internal/worker/migrationmaster/worker.go:	return errors.Annotate(err, "target prechecks failed")
apiserver/facades/client/controller/controller.go:	return errors.Annotate(err, "target prechecks failed")
```

Perhaps this message should be re-written to be more specific to this specific location, so it isn't confused with the worker?

Reply:
Yeah, I'll sweep back around to that in another PR.